graylog2-server/src/main/java/org/graylog2/shared/plugins/PluginLoader.java

```diff
@@ -82,25 +85,27 @@ public PluginLoader(File pluginDir, ChainingClassLoader classLoader) {
         }
 
         LOG.debug("Loading [{}] plugins", files.length);
-        final ImmutableSet.Builder<Plugin> plugins = ImmutableSet.builder();
-        for (File jar : files) {
-            if (!jar.isFile()) {
-                LOG.debug("{} is not a file, skipping.", jar);
-            } else {
-                try {
-                    LOG.debug("Loading <" + jar.getAbsolutePath() + ">");
-                    final ClassLoader pluginClassLoader = URLClassLoader.newInstance(new URL[]{jar.toURI().toURL()});
-                    classLoader.addClassLoader(pluginClassLoader);
-                    final ServiceLoader<Plugin> pluginServiceLoader = ServiceLoader.load(Plugin.class, classLoader);
-
-                    plugins.addAll(pluginServiceLoader);
-                } catch (MalformedURLException e) {
-                    LOG.error("Cannot open JAR file for discovering plugins", e);
-                }
-            }
-        }
-
-        return plugins.build();
+        final List<URL> urls = Lists.newArrayList(files).stream()
```

Comment:
This could simply use [`Arrays.stream(T[] array)`](https://docs.oracle.com/javase/8/docs/api/java/util/Arrays.html#stream-T:A-) instead of instantiating a new array list and then calling `stream()` on it.